packages/rum-core/src/domain/contexts/viewContexts.spec.ts

```diff
@@ -105,6 +154,22 @@ describe('viewContexts', () => {
       lifeCycle.notify(LifeCycleEventType.BEFORE_VIEW_CREATED, buildViewCreatedEvent({ name: 'Fake name' }))
       expect(viewContexts.findView()!.name).toBe('Fake name')
     })
+
+    it('should update the view name for the current context', () => {
+      const { lifeCycle } = setupBuilder.build()
+      lifeCycle.notify(
+        LifeCycleEventType.BEFORE_VIEW_CREATED,
+        buildViewCreatedEvent({
+          id: '0',
+          name: 'foo',
+          startClocks: relativeToClocks(10 as RelativeTime),
+          service: 'test',
+          version: '1',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why do you override all these attribute? Could `buildViewCreatedEvent({ name: 'foo' })` be enough?